packages/tools/devtools/devtools-browser-extension/src/devtools/BackgroundConnection.ts

```diff
@@ -186,3 +186,18 @@ export class BackgroundConnection
 		);
 	};
 }
+
+/**
+ * This functions returns the corresponding tabId based on the testing purposes.
```

Comment:
```suggestion
 * Gets the associated tab's unique ID.
 * @remarks In tests, this can be mocked by setting `TEST_TAB_ID_OVERRIDE` on the window.
```